### PR TITLE
Removed Module Creator from distribution

### LINKED ISSUE #3620: The extension "Module Creator" should not be part of a base DNNplatform install
## Description of problem
The extension "Module Creator" is not needed for the basic DNNPlatform install pack. 

## Description of solution
The extension "Module Creator" should be removed from DNNPlatform and made available as a separate installable extension.

## Description of alternatives considered
None

## Affected browser

* [X] Chrome
* [X] Firefox
* [X] Safari
* [X] Internet Explorer
* [X] Edge

### PR DESCRIPTION
**Pull request by valadas**
- Removes the Module Creator admin module from the distribution as per it's move to https://github.com/DNNCommunity/Dnn.ModuleCreator/releases/tag/v9.8.1
- Also reduces build warnings by about 46

Closes #3620

### CODE REVIEW

**Review by bdukes — COMMENTED** [review]
Is there an issue discussing the removal of this module?  I don't recall discussing this one.

**Comment by valadas** [comment]
Oh sorry, forgot to link the isssue #3620 

Basically the reasoning was that it's something for developers and should not be part of the base install. On the linked repository I took the existing code and fixed all warnings for deprecations and made it a standalone buildable solution. People who do need this module can grab it from there and it makes the base distribution lighter.

**Review by bdukes — APPROVED** [review]

**Comment by valadas** [comment]
Are we good with this in 9.8.1 or we slate it for later (10) ? Just for context, it would not affect upgrades but new installs would not have the ModuleCreator in it.

**Comment by david-poindexter** [comment]
@valadas I am thinking it is good for 9.8.1

**Review by david-poindexter — APPROVED** [review]
Awesome! Thanks @valadas 🎉

**Comment by david-poindexter** [comment]
I put a `Status: On Hold` label on this until we decide whether it is a 9.8.1 or later milestone.

**Review by mitchelsellers — APPROVED** [review]
I'm good with 9.8.1 for this
